libcontainer/container_linux.go

```diff
@@ -581,6 +584,8 @@ func (c *linuxContainer) criuSwrk(process *Process, req *criurpc.CriuReq, opts *
 	defer criuServer.Close()
 
 	args := []string{"swrk", "3"}
+	logrus.Debugf("Using CRIU %d at: %s", c.criuVersion, c.criuPath)
+	logrus.Debugf("Using CRIU with following args: %s", args)
```

Comment:
I could probably loop over the parameters for a nicer print-out. This would increase the criu debug output from three lines to lot of lines. Would be easier to read. My primary goal was to see what is going on. If you want I can try something more verbose.